flow/Histogram.cpp

```diff
@@ -117,7 +118,7 @@ void Histogram::writeToLog() {
 	e.detail("Group", group).detail("Op", op).detail("Unit", UnitToStringMapper.at(unit));
 
 	for (uint32_t i = 0; i < 32; i++) {
-		uint32_t value = ((uint32_t)1) << (i + 1);
+		uint32_t value = i == 31 ? std::numeric_limits<uint32_t>::max() : ((uint32_t)1) << (i + 1);
```

Comment:
Since this is being used to print out an upper bound, could we do the computation with 64-bit integers? I think that gives the correct bounds without any adverse behavior.